src/appleseed/renderer/modeling/camera/orthographiccamera.cpp

```diff
@@ -251,6 +252,14 @@ namespace
             return true;
         }
 
+        RasterizationCamera get_rasterization_camera() const override
+        {
+            RasterizationCamera rc;
+            rc.m_aspect_ratio = 1024.0 / 576.0;
```

Comment:
You're right, for the orthographic camera we can compute the film's aspect ratio (will push a fix) but for the spherical camera we cannot, that's why we return a reasonable camera. All of this will definitely be refined over time.

Reply:
Ah, those are going to be configurable as well. Nice then.